source/_docs/z-wave/adding.markdown

```diff
@@ -38,7 +42,8 @@ Each individual value in the defined key can be anywhere from 0x00 to 0xFF. Defi
 
 ### Network Key
 
-An easy script to generate a random key:
+An easy UNIX script to generate a random key:
```

Comment:
Lets make it Linux, because technically we don't support the full Unix family.
Furthermore, the user might not be aware of the portability of this command to Linux.